packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -104,13 +104,15 @@ export class ProjectService {
         if (user.ability.cannot('create', 'Project')) {
             throw new ForbiddenError();
         }
-        const [adapter, explores] = await ProjectService.testProjectAdapter(
-            data,
-        );
         const projectUuid = await this.projectModel.create(
```

Comment:
Was it intentional to switch the order of these two? It used to test and only save if there was an error. now it creates a project and then tests.

Is it causing an error because we run a full compile? I think we can remove that and only trigger the `refreshAllTables ` after the test + save is successful. Only if it's a problem

Reply:
I needed it for testProjectAdapter. But I agree, let's remove the whole refresh in that method